assets/stylesheets/layout/_main.scss

```diff
@@ -280,6 +280,13 @@ html.iframed {
 	line-height: 1;
 }
 
+/* Emoji and smileys */
```

Comment:
I think so, yes.

I can appreciate heavily scoping CSS as much as the next one — all sorts of good things come from it. But in this particular case, I feel like emoji (and to extend that, smileys) fall under generic typography, and so should be much less scoped. Especially because otherwise we end up with edgecases that have huge smileys or emoji, because they aren't scaled down.